lib/_http_server.js

```diff
@@ -485,8 +485,10 @@ function connectionListener(socket) {
       res.assignSocket(socket);
     }
 
+    socket.setTimeout(self.timeout);
```

Comment:
I agree `self.timeout` should be enough here.

I wish we would enforce any of this styles, but we don't really do it. My preference would be either 3 or 2, but not 1. (If you asked for this)